verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -35,13 +35,29 @@ ABSL_FLAG(bool, printextraction, false,
           "Whether or not to print the extracted general indexing facts "
           "from the middle layer)");
 
-// TODO(ikr): Rename to print_kythe_facts_json
-ABSL_FLAG(bool, printkythefacts, false,
+ABSL_FLAG(bool, print_kythe_facts_json, false,
```

Comment:
Another suggestion, since it is unlikely a user will want to print both formats, so make this a mutually exclusive choice:

```c++
enum class PrintMode { kNoPrint, kJSON, kProto, };
ABSL_FLAG(PrintMode, print_kythe_facts, PrintMode::kNoPrint, "...");
```

See also uses of [EnumFlagMap](https://cs.opensource.google/verible/verible/+/master:common/util/enum_flags.h;bpv=1;bpt=1;l=94?q=enum_flags&ss=verible%2Fverible&gsn=EnumNameMap&gs=kythe%3A%2F%2Fgithub.com%2Fgoogle%2Fverible%3Flang%3Dc%252B%252B%3Fpath%3Dcommon%2Futil%2Fenum_flags.h%23nEm2ekKtJDMYBSDvpAcNcXE6MR_QmkDwkZpiwWRVW8c) to make this easy.